README.md

```diff
@@ -218,6 +218,30 @@ streamingClient.sendAudioData(ByteString);
 
 The streaming connection will close when you call the method `streamingClient.close()` or if you go 15 seconds without sending any audio data.
 
+## Custom Vocabulary (for streaming)
+
+You can now submit any custom vocabularies independently through the new CustomVocabularies client! The main benefit is that users of the SDK can now submit their custom vocabularies for pre-processing and then include these processed custom vocabularies in their streaming jobs.
```

Comment:
I would refrain from using adjectives "new" or "old" to describe entities. These terms don't hold up over more than a short time window. 1 year from now, no one will know the difference between "new" or "old" naming.

Reply:
Strive to write timeless documentation that someone 10 years from now will be able to read.

Reply:
Funny. I pulled this from our other SDKs. I will edit and file some tickets to update the python and node readme's